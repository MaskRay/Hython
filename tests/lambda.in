def factorial(n, s):
    if n < 2: return s
    return factorial(n-1, s * n)

l = lambda n: factorial(n, 1)
print l(36) # my no. when i was at kindergarten
