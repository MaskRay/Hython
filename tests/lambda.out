371993326789901217467999448150835200000000
